app/grandchallenge/challenges/forms.py

```diff
@@ -604,24 +574,58 @@ def clean_status(self):
         return status
 
 
-class ChallengeRequestBudgetUpdateForm(
-    ChallengeRequestBudgetFieldValidationMixin, forms.ModelForm
-):
+class ChallengeRequestBudgetUpdateForm(forms.ModelForm):
+    algorithm_selectable_gpu_type_choices = forms.MultipleChoiceField(
+        # replace NO_GPU value ("") in choices to avoid HTMX POST issue
+        choices=[
+            ("no_gpu", GPUTypeChoices.NO_GPU.label),
```

Comment:
`"no_gpu"` seems to be repeated in a few places. Maybe create a variable:

```python
HTMX_BLANK_CHOICE_KEY = "__HTMX_BLANK_CHOICE_KEY__"
```

and then use that in this class and the tests.

Reply:
I like that. Thanks!